src/spyglass/common/common_region.py

```diff
@@ -45,5 +45,8 @@ def fetch_add(
             subregion_name=subregion_name,
             subsubregion_name=subsubregion_name,
         )
-        cls.insert1(key, skip_duplicates=True)
-        return (BrainRegion & key).fetch1("region_id")
+        query = BrainRegion & key
+        if not query:
+            cls.insert1(key)
+            query = BrainRegion & key
+        return query.fetch1("region_id")
```

Comment:
As far as I can tell, this is equivalent to the former. How does this change the result in your case?

Reply:
The former ended up inserting another entry to the BrainRegion table with a new `region_id`.  This caused an issue with the fetch1 call.   I don't know why the skip_duplicates wasn't applying, my guess is because of how region_id is handled.  

This change reverts back to what the code was before

Reply:
I'm not familiar with the `smallint auto_increment` type uses in the `BrainRegion` table, but I think that may be why `skip_duplicates isn't behaving as intended here